client/app/components/calendar/event/Event.jsx

```diff
@@ -9,11 +9,12 @@ import './event.scss';
 
 export default class Event extends React.Component {
   static propTypes = {
-    event:                    EventSchema.isRequired,
-    containerHeight:          PropTypes.number.isRequired,
+    event: EventSchema.isRequired,
```

Comment:
You need to be consistent - either align always props to right or simple have one space